client/src/utils/validators.js

```diff
@@ -11,16 +12,14 @@ const validateTopLevelDomain = (string) => {
 
 export const validateEmail = (email) => {
   let errorMessage = "";
-  const emailInput = String(email).toLowerCase();
-  const re = /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
   if (!email) {
-    errorMessage = "emailRequired";
-  } else if (email.length > 200) {
-    errorMessage = "emailMaxLength";
-  } else if (!re.test(emailInput)) {
-    errorMessage = "emailInvalid";
-  } else if (!validateTopLevelDomain(emailInput)) {
-    errorMessage = "emailDomainInvalid";
+    errorMessage = "Email address is required";
+  } else if (email.length > 254) {
+    errorMessage = "Email address must not exceed 254 characters";
+  } else if (!_isEmail(email)) {
+    errorMessage = "Email address is invalid";
+  } else if (!validateTopLevelDomain(email)) {
+    errorMessage = "Email address is invalid";
```

Comment:
```suggestion
  } else if (!_isEmail(email) || !validateTopLevelDomain(email)) {
    errorMessage = "Email address is invalid";
```

Let's combine this since the message is the same

Reply:
Okay I resolved this but will ask a few questions.